lib/prometheus/active_record.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Yabeda.configure do
+  group :rails_connection_pool do
+    size = gauge :size, comment: 'Size of the connection pool'
+    connections = gauge :connections, comment: 'Number of connections in the connection pool'
```

Comment:
```suggestion
    connections = gauge :connections, comment: 'Number of connections in the connection pool', tags: %i[state]
```

And you'll able to use the labels again to separate "busy", "dead", "idle".